mimic/test/test_ironic.py

```diff
@@ -45,19 +45,114 @@ def setUp(self):
             "power_state", "instance_info", "ports", "name", "driver_internal_info",
             "inspection_finished_at", "inspection_started_at", "clean_step"
         ]
+        self.url = "/ironic/v1/nodes"
+        self.create_request = {
+            "chassis_uuid": str(uuid4()),
+            "driver": "fake",
+            "driver_info": {"cache_image_id": None,
```

Comment:
I would not add the caching values here. In fact, for driver_info, typically all you want when creating is creds to use the driver (in the case of fake driver, I think an empty dict is fine).